tests/telemetry/test_telemetry.py

```diff
@@ -38,8 +38,8 @@ def test_telemetry_enabled_on_first_tracer_flush(test_agent_session, ddtrace_run
     # No trace and No Telemetry
     assert len(test_agent_session.get_events()) == 0
 
-    # Submit a trace to the agent in a subprocess
-    code = 'from ddtrace import tracer; span = tracer.trace("test-telemetry"); span.finish()'
+    # Submit a trace to the agent in a subprocess and imports flask to generate an integrations changed event
+    code = 'import flask; from ddtrace import tracer; span = tracer.trace("test-telemetry"); span.finish()'
```

Comment:
Importing flask here generates an integrations-changed event.  This is required to fix a failing instrumentation telemetry event. An alternative here is to just refactor this test so it's not reliant on what integrations are imported/patched on application start up.